### PR TITLE
Fixes interactive mode for cvss v4.0

### PR DESCRIPTION
**Pull request by chrisdlangton**
addresses #47

### CODE REVIEW

**Comment by chrisdlangton** [comment]
FYI
No tests existed for interactive mode, for me to update.
Searched for imports using `repo:RedHatProductSecurity/cvss path:/^tests\// interactive`

**Review by jsvob — CHANGES_REQUESTED** [review]
The calculator doesn't return a CVSS v4.0 score in either non-interactive and interactive mode.

```
$ cvss_calculator -4 -v "CVSS:4.0/AV:P/AC:H/AT:P/PR:H/UI:A/VC:L/VI:L/VA:L/SC:L/SI:L/SA:L/S:N"
Traceback (most recent call last):
  File "/TMP/cvss/venv/bin/cvss_calculator", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/TMP/cvss/venv/lib64/python3.11/site-packages/cvss/cvss_calculator.py", line 63, in main
    scores = cvss_vector.scores()
             ^^^^^^^^^^^^^^^^^^
AttributeError: 'CVSS4' object has no attribute 'scores'

```

**Comment by chrisdlangton** [comment]
image
Should be working now, CVSS4 methods weren't API backwards compatible

File: cvss/cvss4.py [hunk 1]
```diff
@@ -608,6 +608,32 @@ def compute_severity(self):
...
+        Returns:
+            (tuple): Base Severity as string
+        """
+        return (self.severity,)
+
+    def rh_vector(self):
+        """
+        Returns cleaned vector with score in Red Hat notation, e.g. score/vector.
+
+        Example: 6.5/CVSS:3.0/AV:P/AC:H/PR:H/UI:R/S:C/C:H/I:H/A:N/E:H/RL:O/RC:R/CR:H/MAC:H/MC:L
```
Comment (jsvob) [inline cvss/cvss4.py:633]:
```suggestion
        Example: 7.3/CVSS:4.0/AV:P/AC:H/AT:N/PR:H/UI:P/VC:H/VI:H/VA:H/SC:H/SI:H/SA:H/E:A
```

**Review by jsvob — APPROVED** [review]
LGTM with a minor nitpick. Thank you!

**Pull request merged**